.knip.json

```diff
@@ -0,0 +1,31 @@
+{
+  "ignore": [
+    "**/*/fuels.config.*",
+    "**/*/.vitepress/*",
+    "/apps/docs/*",
+    "/packages/abi-typegen/test/**",
+    "templates/**"
+  ],
+  "ignoreDependencies": [
```

Comment:
I'd like to have a more optimal solution than this, which wouldn't ignore _all_ the use cases of `@fuel-ts` for instance but I'd have to know which packages require it without importing it. I think that can be done in a follow up PR